scripts/lastspoke.pl

```diff
@@ -20,11 +20,14 @@
 # Triggers on !lastspoke <nick>, !seen <nick> and !lastseen <nick>
 # 
 use strict;
+use utf8;
+use Encode qw/decode encode/;
 use Irssi;
 use Irssi::Irc;
+use JSON::PP;
```

Comment:
Requires installing of extra module perl-JSON-PP, at least on my fedora 31 server system. I installed it and am now testing this script.